notes.txt

```diff
@@ -1,11 +1,30 @@
 Some notes on mountains.
 
 Mountains in Snowdonia:
-Snowden
+Snowdon
 Cadair Idris
 Various
 Others
-asdf
+Asdf
 Everest
-words
-asdf
+Words
+Pen y Pass
+Crib Goch
+Garnedd Ugain
+Derlwyn
+Elidir Fawr
+Y Garn
+Glyder Fawr
+Glyder Fach
+Tryfan
+Ogwen
+Pen yr Ole Wen
+Carnedd Dafydd
+Carnedd Llewellyn
+Yr Elen
+Carnedd Llewellyn
```

Comment:
@Fedjmike You have a duplicate here. Note also that the preferred spelling would be "Llewelyn", as the mountain is named for Llywelyn ap Gruffudd, once King of Wales. (Great Pull Request otherwise!)